website/project/model.py

```diff
@@ -506,9 +506,16 @@ def validate_title(value):
     if value is None or not value.strip():
         raise ValidationValueError('Title cannot be blank.')
 
+    value = sanitize.strip_html(value)
+
+    if value is None or not value.strip():
+        raise ValidationValueError('Invalid title.')
+
     if len(value) > 200:
         raise ValidationValueError('Title cannot exceed 200 characters.')
-    return True
+
+    return value
```

Comment:
Thanks for keeping sanitization separate; that'll make any further validation cleanup easier in the future (a different PR). :rabbit2: